readme.md

```diff
@@ -317,6 +300,57 @@ print(UserDefaults.standard.bool(forKey: Defaults.Keys.isUnicornMode.name))
 //=> true
 ```
 
+### Custom types
+
+Although `Defaults` already support many types internal, there might have some situations that you want to use your own type.
+The guide below will show you how to create your own custom type.
```

Comment:
You are not creating a custom type here, you're really showing how to make a custom type work with `Defaults`.